dashboard/src/components/ServiceInstanceList/ServiceInstanceList.tsx

```diff
@@ -93,6 +93,50 @@ class ServiceInstanceList extends React.PureComponent<
       pushSearchFilter,
     } = this.props;
     const loaded = !brokers.isFetching && !instances.isFetching && !classes.isFetching;
+    let body = <span />;
+    if (!isServiceCatalogInstalled) {
```

Comment:
this change is required in #719 but I'm adding it here since the catalog reducer changes are more complete in this PR (sorry!)